sdk/keyvault/azure-keyvault-keys/tests/test_key_client.py

```diff
@@ -427,6 +429,61 @@ def test_get_random_bytes(self, client, **kwargs):
             assert all([random_bytes != rb] for rb in generated_random_bytes)
             generated_random_bytes.append(random_bytes)
 
+    @only_hsm_7_3_preview()
+    @client_setup
+    def test_key_release(self, client, **kwargs):
+        attestation_uri = self._get_attestation_uri()
+        attestation = get_attestation_token(attestation_uri)
+        release_policy = get_release_policy(attestation_uri)
+
+        rsa_key_name = self.get_resource_name("rsa-key-name")
+        key = self._create_rsa_key(
+            client, rsa_key_name, hardware_protected=True, exportable=True, release_policy=release_policy
+        )
+        assert key.release_policy
+        assert key.release_policy.data
+        assert key.properties.exportable
+
+        release_result = client.release_key(rsa_key_name, attestation)
+        assert release_result.value
+
+    @only_hsm_7_3_preview()
+    @client_setup
+    def test_imported_key_release(self, client, **kwargs):
+        attestation_uri = self._get_attestation_uri()
+        attestation = get_attestation_token(attestation_uri)
+        release_policy = get_release_policy(attestation_uri)
+
+        imported_key_name = self.get_resource_name("imported-key-name")
+        key = self._import_test_key(
+            client, imported_key_name, hardware_protected=True, exportable=True, release_policy=release_policy
+        )
+        assert key.release_policy
+        assert key.release_policy.data
+        assert key.properties.exportable
+
+        release_result = client.release_key(imported_key_name, attestation)
+        assert release_result.value
+
+    @only_hsm_7_3_preview()
+    @client_setup
+    def test_exportable_without_release_policy_error(self, client, **kwargs):
+        """A key cannot be created as exportable without having a release policy"""
+        key_name = self.get_resource_name("key-name")
+        with pytest.raises(HttpResponseError):
```

Comment:
Why test the service's argument validation?

Reply:
There's not really a compelling reason to, other than that JS tests for this and I was curious about what behavior is allowed. You're right, though -- this doesn't have anything to do with SDK code and is superfluous. I'll use this real estate for more valuable tests